mathics/builtin/numbers/numbertheory.py

```diff
@@ -476,7 +476,8 @@ def to_int_value(x):
         result = n.to_python()
         for i in range(-py_k):
             try:
-                result = sympy.ntheory.prevprime(result)
+                # from sympy 1.13, the previous prime to 2 fails...
```

Comment:
Sympy v1.13 seems to have problems to find out that the previous prime to 2 is -2